src/test/testing-library/ForWhoSection.test.tsx

```
@@ -1,4 +1,4 @@
-import { render } from '@testing-library/react';
+import { render, within } from '@testing-library/react';
```

ESLint

[error] 'within' is defined but never used.

(@typescript-eslint/no-unused-vars)